services/CryptoGetAccountBalance.proto

```diff
@@ -62,15 +62,3 @@ message CryptoGetAccountBalanceResponse {
     uint64 balance = 3; // The current balance, in tinybars.
     repeated TokenBalance tokenBalances = 4; // The token balances possessed by the target account.
 }
-
```

Comment:
Should we move TokenBalance and TokenBalances to BasicTypes so stream file doesn't have to depend upon CryptoGetAccountBalance.proto? I don't think this changes the generated classes, but not sure about non-java stuff.

Reply:
Hmm yes you're right, I compiled both ways and it doesn't seem to matter for Java at least.

Moved them, can see if anyone complains.